lib/fluent/plugin/out_http.rb

```diff
@@ -100,11 +102,25 @@ def initialize
       @uri = nil
       @proxy_uri = nil
       @formatter = nil
+
+      @Cache = []
+      @CacheIdMutex = Mutex.new
+      @CacheEntry = Struct.new(:uri, :conn)
```

Comment:
We use snake case for instance variables.
```suggestion
      @cache = []
      @cache_id_mutex = Mutex.new
      @cache_entry = Struct.new(:uri, :conn)
```

Reply:
I renamed the instance variables.